src/components/GamePage/GamePage.js

```diff
@@ -1,25 +1,56 @@
 import { Link } from 'react-router-dom'
+import { useState } from 'react'
 import styled from 'styled-components/macro'
 import Player from '../Player'
 import Button from '../Button'
+import ShowWinner from '../ShowWinner'
+
+export default function GamePage({ players, onScore, resetPlayers, winners }) {
+  const [visible, setVisible] = useState('whilePlaying')
 
-export default function GamePage({ players, onScore, resetPlayers }) {
   return (
-    <WrapperGamePage>
-      <h1>Score!</h1>
-      {players.map(({ name, score }, index) => (
-        <Player
-          key={index}
-          player={name}
-          score={score}
-          onScore={() => onScore(index)}
-          disabled={players.length <= 1}
-        />
-      ))}
-      <Button as={Link} to="/" onClick={resetPlayers}>
-        End game
-      </Button>
-    </WrapperGamePage>
+    <>
+      {visible === 'whilePlaying' && (
+        <WrapperGamePage>
+          <h1>Score!</h1>
+          {players.map(({ name, score }, index) => (
+            <Player
+              key={index}
+              player={name}
+              score={score}
+              onScore={() => onScore(index)}
+              disabled={players.length <= 1}
+            />
+          ))}
+          <Button as={Link} to="/" onClick={resetPlayers}>
+            End game
+          </Button>
+
+          <Button
+            visible={visible}
+            onClick={() => {
+              setVisible('winner')
+            }}
+          >
+            Show winner!
+          </Button>
+        </WrapperGamePage>
+      )}
+
+      {visible === 'winner' && (
+        <>
+          <ShowWinner
+            hidden={visible}
+            title={'Winner'}
+            players={players}
+            resetPlayers={resetPlayers}
```

Comment:
This shouldn't work as you don't have the props `resetPlayers` in your `ShowWinner` component. Also, have a look the the props you defined in the component! You should set: `{ visible, title, players, confetti }` although you are never using `confetti`. Here you are setting `{ hidden, title, players, resetPlayers }`. The props need to have the same name :)